### PR TITLE
Fix some warnings in MediaBrowser.Controller

### PR DESCRIPTION
**Pull request by Bond-009**
#2149

### CODE REVIEW

File: MediaBrowser.Controller/Entities/PersonInfo.cs [hunk 1]
```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using MediaBrowser.Model.Entities;
+
+namespace MediaBrowser.Controller.Entities
+{
+    /// <summary>
+    /// This is the small Person stub that is attached to BaseItems
```
Comment (dkanada) [inline MediaBrowser.Controller/Entities/PersonInfo.cs]:
```suggestion
    /// This is a small Person stub that is attached to BaseItems.
```

File: MediaBrowser.Controller/Entities/PersonInfo.cs [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        public string Role { get; set; }
+
+        /// <summary>
+        /// Gets or sets the type.
+        /// </summary>
+        /// <value>The type.</value>
+        public string Type { get; set; }
+
+        /// <summary>
+        /// Gets or sets the sort order - ascending
```
Comment (dkanada) [inline MediaBrowser.Controller/Entities/PersonInfo.cs]:
```suggestion
        /// Gets or sets the ascending sort order.
```

File: MediaBrowser.Controller/Entities/PersonInfo.cs (hunk 1 above)
Comment (Bond-009) [reply MediaBrowser.Controller/Entities/PersonInfo.cs]: Why not "the" when it only one?

Comment (dkanada) [reply MediaBrowser.Controller/Entities/PersonInfo.cs]: There's only one per person but many in the database right?

Comment (Bond-009) [reply MediaBrowser.Controller/Entities/PersonInfo.cs]: 🤷‍♂

**Pull request merged**